package.json

```diff
@@ -34,6 +34,7 @@
     "prepare": "husky"
   },
   "devDependencies": {
+    "@types/uuid": "^9.0.8",
```

Comment:
Looks like this was added by mistake here :)

LPT: it's a good idea to review the code/PR yourself before assigning other reviewers - I imagine this would've been caught by yourself too.

Reply:
Yes indeed sorry for that. I removed it.